cmd/kata-local-ci

```diff
@@ -0,0 +1,260 @@
+#!/bin/bash
+#
+## Usage: kata-local-ci <options>
+##   Run or setup local CI environment for kata containers using libvirt.
+##
+## Options include:
+##   -h: Print help
+##   -d <distro>: Select VM distro (fedora,ubuntu,centos, default:fedora)
+##   -j <job>: CI job to run (default: all)
+##   -w <dir>: Work directory to test (default $GOPATH/src/github.com/kata-containers)
+##   -f: force install of the VM (and overwrite disk)
+##   -F: "Faster" by reusing the image without reverting to earlier snapshot
+##
+## Creation options include:
+##   -k <ssh_key_file> (default ~/.ssh/id_rsa.pub)
+##   -o <disk-image>: Path and image name for the VM
+##   -s <size>: Disk image size for the VM (default 50G)
+##   -m <memory>: Memory for the VM in megabytes (default 8192M)
+##   -c <cpus>: Number of CPUs for the VM (default 4)
+##   -n network config (optional)
+##
+
+distro=fedora
+size=50G
+memory=8192
+cpus=4
+key=~/.ssh/id_rsa.pub
+vmdir=go/src/github.com/kata-containers
+directory=${GOPATH:-~go}/src/github.com/kata-containers
+disk=""
+network=""
+job=""
+kci=~/.kata-containers-ci
+install=false
+revert=true
+
+function help {
+    sed -E '/^##/s/^##\s?//g;t;d' $0
+}
+
+function progress() {
+    printf "******************************************** [$(date +%H:%M:%S)]\n"
+    printf "*\n"
+    printf "* $1\n"
+    printf "*\n"
+    printf "********************************************\n"
+}
+
+function sys() {
+    echo "# sudo" "$@"
+    sudo "$@"
+}
+
+function run() {
+    echo "%" "$@"
+    "$@"
+}
+
+function ci_cmd() {
+    echo "$ ssh kata@$VM_IPADDR -o StrictHostKeyChecking=no -o UpdateHostKeys=no" "\"$@\""
+    ssh kata@$VM_IPADDR -o StrictHostKeyChecking=no -o UpdateHostKeys=no "$@"
+}
+
+function ci_root_cmd() {
+    echo "$ ssh root@$VM_IPADDR -o StrictHostKeyChecking=no -o UpdateHostKeys=no" "\"$@\""
+    ssh root@$VM_IPADDR -o StrictHostKeyChecking=no -o UpdateHostKeys=no "$@"
```

Comment:
You could remove the duplication here with something like:

```
ci_root_cmd()
{
    local cmd="ssh root@$VM_IPADDR -o StrictHostKeyChecking=no -o UpdateHostKeys=no" "\"$@\""
    echo "$cmd"
    eval $cmd
}
```

Same comment for `sys()` and `run()`.